cts/scheduler/exp/inc12.exp

```diff
@@ -48,9 +48,7 @@
     <action_set>
       <rsc_op id="26" operation="stop" operation_key="heartbeat_192.168.100.182_stop_0" on_node="c001n02" on_node_uuid="e8973f32-1c97-40cb-a84a-c489f49664a5">
         <primitive id="heartbeat_192.168.100.182" class="ocf" provider="heartbeat" type="IPaddr"/>
-        <attributes CRM_meta_on_node="c001n02" CRM_meta_on_node_uuid="e8973f32-1c97-40cb-a84a-c489f49664a5" CRM_meta_timeout="10000" >
-          <param name="1" value="192.168.100.182"/>
-        </attributes>
+        <attributes 1="192.168.100.182" CRM_meta_on_node="c001n02" CRM_meta_on_node_uuid="e8973f32-1c97-40cb-a84a-c489f49664a5" CRM_meta_timeout="10000" />
```

Comment:
I believe the controller would be unable to parse this into an xmlNode

Reply:
Surprised cts-scheduler wouldn't hit an issue, but if it never consumes the graph (or whatever the exp files are), then I guess that makes sense.

Reply:
Exactly, the scheduler produces transition graphs for the controller to use